entity-framework/core/what-is-new/ef-core-2.1.md

```diff
@@ -0,0 +1,69 @@
+
+- Storing enums as strings
+- Mapping unsigned integers with SQL Server
+- Automatic encryption and decryption of property values
+
+## LINQ GroupBy translation
+Before EF Core 2.1, the GroupBy LINQ operator would always be evaluated in memory. We now support translating it to the SQL GROUP BY clause in most common cases.
+
+## Data Seeding
+With the new release it will be possible to provide initial data to populate a database. Unlike in EF6, in EF Core, seeding data is associated to an entity type as part of the model configuration. Then EF Core migrations can automatically compute what insert, update or delete operations need to be applied when upgrading the database to a new version of the model.
+
+## Query types
+An EF Core model can now include query types. Unlike entity types, query types do not have keys defined on them and cannot be inserted, deleted or updated (i.e. they are read-only), but they can be returned directly by queries. Some of the usage scenarios for query types are:
+
+- Mapping to views without primary keys
+- Mapping to tables without primary keys
+- Mapping to queries defined in the model
+- Serving as the return type for FromSql() queries
+
+## Include for derived types
+It will be now possible to specify navigation properties only defined in derived types when writing expressions for the Include() methods. We support both an explicit cast and the `as` operator:
+
+ ``` csharp
+ var queryUsingExplicitCast = context.People.Include(p => ((Student)p).School);
+ var queryUsingAsOperator = context.People.Include(p => (p as Student).School);
+```
+
+## System.Transactions support
+We have added the ability to work with System.Transactions features such as TransactionScope. This will work on both .NET Framework and .NET Core when using database providers that support it.
+
+## Better column ordering in initial migration
+Based on customer feedback, we have updated migrations to initially generate columns for tables in the same order as properties are declared in classes. Note that we cannot change order when new members are added after the initial table creation.
+
+## Optimization of correlated subqueries
+We have improved our query translation to avoid executing N + 1 SQL queries in many common scenarios in which a root query is joined with a correlated subquery. The optimization requires buffering results and you have to explicitly opt-in is a change to the current behavior to  streaming, so it is not enabled by default. In order to opt-into the optimization
```

Comment:
we don't support explicit joins - just ones that we create as result of navigation rewrite

Reply:
also - "behavior to  streaming" has double space between "to" and "streaming"